src/components/EarnDashboard/components/PositionDetails/StakingPositionsByProvider.tsx

```diff
@@ -85,7 +85,9 @@ export const StakingPositionsByProvider: React.FC<StakingPositionsByProviderProp
   const stakingOpportunities = useAppSelector(
     selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
   )
-  const filteredDown = stakingOpportunities.filter(e => ids.includes(e.assetId as OpportunityId))
+  const filteredDown = stakingOpportunities.filter(
+    e => ids.includes(e.assetId as OpportunityId) || ids.includes(e.id as OpportunityId),
+  )
```

Comment:
Should live at selector-level, will author as a follow-up to keep changes minimal here